CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 * build: remove unused implicit features from cairo-vm
 
+* build: remove dependecy to `thiserror` (use `thiserror-no-std/std` instead)
```

Comment:
I think this is on an old version of the changelog. We should merge main to this PR and move this again to the "Upcoming Changes" section.

Reply:
I just fixed it :)